### DISCUSSION TITLE
[designer] css/codicon.tff not found after webpack build

### DISCUSSION THREAD

**Discussion by becory** [microsoft/AdaptiveCards#8293] [Q&A]
Hi, @anna-dingler 

How can I import the CSS in webpack?

I already add in my main.js, but it seems doesn't work for me.
`import 'adaptivecards-designer/dist/adaptivecards-designer.css';`

![Screen Shot 2023-02-08 at 11 05 06 AM](https://user-images.githubusercontent.com/13202145/217419786-a531677a-8589-427e-868d-64c7538aeada.png)
![Screen Shot 2023-02-08 at 11 08 03 AM](https://user-images.githubusercontent.com/13202145/217419790-6a271db1-2304-43df-b22d-4477d46ea587.png)

**Comment by anna-dingler** [comment]
Hi @becory,

Can you try using the `CopyWebpackPlugin` to move the CSS to the expected location?
